### PR TITLE
Refactoring map and feed logic

### PR DESCRIPTION
**Pull request by Twissi**
Splits map and feed logic into separate objects

### CODE REVIEW

File: source/javascripts/app/feed.js [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
+var Feed = {
+
+  settings: {
+    maxItemsCount: 6,
+    feedUrl: "https://weltreise.abwesend.com/s3/weltreise-log.json",
+    feedDomEle: document.getElementById("feedJson")
+  },
+
+  init: function(){
+    console.log("init feed");
```
Comment (beanieboi) [inline source/javascripts/app/feed.js]: I think we should remove the debugging statements

Comment (beanieboi) [inline source/javascripts/app/feed.js]: I think we should remove the debugging statements

File: source/javascripts/app/map.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        url: "/images/location_red.png", // url
+        scaledSize: new google.maps.Size(35, 47), // scaled size
+        origin: new google.maps.Point(0, 0), // origin
+        anchor: new google.maps.Point(17, 47) // anchor
+      }
+    }
+  },
+
+  init: function() {
+    console.log("init map");
```
Comment (beanieboi) [inline source/javascripts/app/map.js]: same here

**Review by beanieboi — CHANGES_REQUESTED** [review]
just two minor comments, it looks great! :)

**Comment by Twissi** [comment]
@beanieboi Updated PR

**Review by beanieboi — APPROVED** [review]

**Pull request merged**